### PR TITLE
add rating components , moviecard , styling

### PR DESCRIPTION
**Pull request by azmat239**

### CODE REVIEW

**Review by auqibyousuf — APPROVED** [review]

File: app/components/Rating/Rating.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { RatingTypes } from "./Rating.types";
+import classNames from "classnames";
+
+const Rating = ({ value, variant, extraClasses }: RatingTypes) => {
+  const ratingClasses = classNames({
+    "w-10 bg-lightBlack rounded-full h-10 text-white text-center p-2 text-sm font-bold absolute top-[200px] left-8 border-2":
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.tsx]: the absolute styles should not be added in the component like this, instead you should pass these as extraClasses.
Reason: This component might be used somewhere else where this posirion is not required

File: app/components/Rating/Rating.stories.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  tags: ["autodocs"],
+  parameters: {
+    layout: "fullscreen",
+  },
+} satisfies Meta<typeof Rating>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Rating_Card: Story = {
```
Comment (UmmarHamid) [inline app/components/Rating/Rating.stories.ts]:
```suggestion
export const Movie_Rating: Story = {
```

File: app/components/MenuPopover/MenuPopover.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            return (
+              <Link href={link.url} className="w-full p-2 hover:cursor-pointer">
+                {link.text}
+              </Link>
+            );
+          })}
+        </div>
+        <Popover.Close className="absolute top-2 right-2">
+          <GiCrossedBones />
+        </Popover.Close>
```
Comment (UmmarHamid) [inline app/components/MenuPopover/MenuPopover.tsx]: This is  probably not required as clicking outside should close the menu

File: app/components/MovieCard/MovieCard.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        src={image.imgSrc}
+        alt={image.altText}
+        width={150}
+        height={220}
+        className="rounded-8"
+        onClick={onClick}
+      />
+      <MenuPopover
+        links={MenuPopoverLinks}
+        extraClasses="absolute top-2 left-36"
```
Comment (UmmarHamid) [inline app/components/MovieCard/MovieCard.tsx]:
```suggestion
        extraClasses="absolute top-2 right-[someValue]"
```

**Review by UmmarHamid — CHANGES_REQUESTED** [review]
Look good 
Small feedack

**Review by UmmarHamid — APPROVED** [review]

**Pull request merged**